src/plugins/intel_cpu/src/nodes/deconv.cpp

```diff
@@ -1117,7 +1203,43 @@ bool Deconvolution::canFuseBias() const {
     //ONEDNN convolution_data_bwd_t can't support bias fusing.
     //Current only int8 precision choose deconvolution_fwd_t.
     return  (canBeExecutedInInt8() &&
-            (externOutShape ? getParentEdges().size() == 3 : getParentEdges().size() == 2));
+            (deconvAttrs.externOutShape ? getParentEdges().size() == 3 : getParentEdges().size() == 2));
+}
+
+void Deconvolution::initSupportedPrimitiveDescriptors() {
+    if (!useACL) {
+        Node::initSupportedPrimitiveDescriptors();
+        return;
+    }
+
+    auto& creatorsMap = BlockedDescCreator::getCommonCreators();
+    auto pushDesc = [&](LayoutType format) {
+        NodeConfig config;
+        config.inConfs.resize(getParentEdges().size());
+        config.outConfs.resize(getOriginalOutputsNumber());
+
+        for (size_t i = 0; i < getParentEdges().size(); ++i) {
+            config.inConfs[i].setMemDesc(
+                    creatorsMap.at(format)->createSharedDesc(getOriginalInputPrecisionAtPort(i), getInputShapeAtPort(i)));
+        }
+        config.outConfs[0].setMemDesc(
+                creatorsMap.at(format)->createSharedDesc(getOriginalOutputPrecisionAtPort(0), getOutputShapeAtPort(0)));
+
+        std::vector<MemoryDescPtr> srcMemoryDescs;
+        for (size_t i = 0; i < config.inConfs.size(); i++) {
+            srcMemoryDescs.push_back(config.inConfs[i].getMemDesc());
+        }
+        std::vector<MemoryDescPtr> dstMemoryDescs;
+        for (size_t i = 0; i < config.outConfs.size(); i++) {
+            dstMemoryDescs.push_back(config.outConfs[i].getMemDesc());
+        }
+
+        auto factory = std::make_shared<DeconvExecutorFactory>(deconvAttrs, srcMemoryDescs, dstMemoryDescs,
+                                                               std::make_shared<ExecutorContext>(context, getImplPriority()));
+
+        supportedPrimitiveDescriptors.emplace_back(config, impl_desc_type::acl, factory);
+    };
+    pushDesc(LayoutType::ncsp);
```

Comment:
BTW, have you tried `nspc` layout?

Reply:
@dmitry-gorokhov added

Reply:
Have you checked correctness locally? As I remember nspc instances are not enabled for ARM by default

Reply:
@dmitry-gorokhov on mac is ignoring and not working on rpi, I removed this layout